.github/workflows/build-test.yaml

```diff
@@ -474,3 +474,26 @@ jobs:
           kubectl -n $APP_NAME logs -p deployment/kotsadm-operator
           kill $ADMIN_CONSOLE_PID
           exit $EXIT_CODE
+
+
+  validate-kots-pull:
```

Comment:
this doesn't validate the result, but it's better than nothing

Reply:
i'm gonna add kubectl apply at least